Release Notes 14.0 (#15335)
opf/openproject: docs/release-notes/14-0-0/README.md

![Work-based progress reporting](progress-work-estimates-workMode.jpg)

#### In status-based progress reporting, Remaining work will be automatically calculated.

In Status-based progress reporting mode, Work is not a required value. However, if Work is set, Remaining work is automatically calculated. To make this link clear and transparent, clicking on any of the three values to modify them will display the following pop-over:

![Status-based progress reporting](progress-work-estimates-statusMode.jpg)

#### Administrators will be able to exclude certain work packages from hierarchy totals of Work, Remaining work and % Complete.

Admins are able to exclude specific work packages (e.g., those marked as rejected) from the total sum calculations of their parent. In this case, a small info icon will appear next to excluded values:

![Progress work estimates excluded from parent](progress-work-estimates-excludedFromParent.jpg)

> In addition to these changes, the section 'Estimates and Time' has been renamed to 'Estimates and Progress' and this is where you will now find % Complete. Also, the seeding of statuses has been fixed to include % Complete values and in the Progress modal, you will now be able to live preview changes amongst the fields.

Reviewer: ```suggestion
In addition to these changes, the section 'Estimates and Time' has been renamed to 'Estimates and Progress' and this is where you will now find % Complete. Also, the seeding of statuses has been fixed to include % Complete values and in the Progress modal, you will now be able to preview changes amongst the fields live.
```

Please note that regarding progress reporting, **updating to OpenProject 14.0 might result in automated data modification in certain cases**. See our blog to learn about the [details and motives on these significant changes to progress and work estimates](https://www.openproject.org/blog/changes-progress-work-estimates/) and how it might effect you.

### Project attributes management on the project overview page

> With OpenProject 14.0, you are now able to view and edit custom fields for projects in a structured way, organized in sections, on the project overview section. These fields are now referred to as 'Project attributes'. Admins can edit them directly on the project overview page:

Reviewer: ```suggestion
With OpenProject 14.0, you are now able to view and edit custom fields for projects in a structured way, organized in sections, on the project overview page. These fields are now referred to as 'Project attributes'. Admins can edit them directly on the project overview page:
```

![Project attributes shown on project overview page, divided in sections](14-0-project-attributes-sections-highlighted.png)

Project attributes as well as the sections can be managed in administration/projects/project attributes.

![Manage project attributes and sections in adminstrations](14-0-project-attributes-administration.png)

> ### Streamlined view of custom fields in project list and project overview

Reviewer: ```suggestion
### Streamlined view of custom fields in project lists and project overview
```

Until now, there have been uncertainties with project-specific custom fields from time to time. This is why the tab 'project' has been removed from the custom fields admin settings pages. Instead, project custom fields are now edited within their own admin settings page. Also, the custom fields widget has been removed. 

As described above, project custom fields are now called 'Project attributes' and can be shown in a sidebar on the project overview page on a per-project configuration.

### Enhanced Meetings module with new features

OpenProject's [(Dynamic) Meetings](https://www.openproject.org/docs/user-guide/meetings/dynamic-meetings/) have been updated with the following new features:

> * Attachments of agenda items are now shown in the sidebar. You can also add them directly to the meeting there.

Reviewer: ```suggestion
* Meetings attachments are now shown in the bottom section of the right-hand sidebar. You can add them directly to the meeting there and drag and drop them from this section to the Notes section of the agenda items.
```

* When copying a meeting, you can now check a box to choose whether you want to include attachments.
* A meeting history has been added and can be found in the Meetings menu, showing previous changes.
* The presenter of an agenda item can now be changed from the creator of the agenda item to any person in the project.
* An option to copy an URL of a specific agenda item has been added.

### Automatic alerts for unhealthy file storages, and toggle options

With OpenProject 14.0, admins now get email notifications when a file storage (e.g Nextcloud) is unhealthy. To avoid sending unwanted messages, we also added the option to deactivate these health status notifications for a storage.

Read more about file storage troubleshooting in our [documentation](https://www.openproject.com/docs/system-admin-guide/file-storages/file-storage-troubleshooting/).

> ### OneDrive/SharePoint: Copying template projects including automatically managed project folder

Reviewer: ```suggestion
### OneDrive/SharePoint: Copying template projects including automatically managed project folders
```

> If you add an automatically managed project folder to a project, you will now have the following options:

Reviewer: ```suggestion
If you copy a project where a file storage has automatically managed project folders selected, you will now have the following options:
```

* deactivate the copy option "File Storages: Project folders" – copying will copy the file storage, but not the specific project folder
* deactivate the copy option "File Storages" – copying will not copy any file storage

### Reminder for admins when revoking a project membership to remove shares on work packages 

> As of 14.0, admins are asked whether they also want to remove access to shared work packages when deleting a project member. 

Reviewer: ```suggestion
As of 14.0, admins are asked whether they also want to remove access to shared work packages when removing a project member or group from a project. 
```
